src/config.rs

```diff
@@ -27,6 +27,15 @@ pub struct Notifications {
     use_fuzzy_matching: bool,
 }
 
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub enum DisplayMode {
+    Everything,
+    ByOutput,
+    ByWorkspace,
+    WorkspaceButtons,
```

Comment:
I think this should be a separate boolean option, because semantically it's still ByWorkspace - but you're adding buttons for each workspace.

Reply:
Hmm something like "display_workspace_buttons" option maybe?

Reply:
Actually only byworkspace doesn't make sense, but still you might want to visualize the workspace you're currently on. So:
 * for everything - it does make sense, but it's currently unsupportable by the current configuration data structure (the code can do it)
 * for by_output - still makes sense - that's what it was originally for
 * for by_workspace - _could_ make sense if it always displays current workspace on output - gives an indication of what the output name is.